src/frontend/Ast_to_Mir.ml

```diff
@@ -257,8 +257,8 @@ let constraint_forl = function
 
 let extract_transform_args = function
   | Program.Lower a | Upper a -> [a]
-  | Offset a -> [{a with Expr.Fixed.pattern= Lit (Int, "0")}; a]
-  | Multiplier a -> [a; {a with pattern= Lit (Int, "1")}]
+  | Offset a -> [a; {a with Expr.Fixed.pattern= Lit (Int, "1")}]
+  | Multiplier a -> [{a with pattern= Lit (Real, "0.0")}; a]
```

Comment:
Why is one `Int` and the other `Real`? It's not like the backend cares.

Reply:
Good call. Changing to Int 0 and merging. Thanks!